app/electron/main.js

```diff
@@ -74,13 +111,12 @@ async function createWindow() {
 
   // Sets up main.js bindings for our electron store;
   // callback is optional and allows you to use store in main process
-  const callback = (/* success, initialStore */) => {
-    /* !TODO LOG
-    console.log(
+  // eslint-disable-next-line func-names
+  const callback = function (success, initialStore) {
+    log.info(
       `${!success ? "Un-s" : "S"}uccessfully retrieved store in main process.`
     );
-    console.log(initialStore); // {"key1": "value1", ... }
-    */
+    log.info(initialStore); // {"key1": "value1", ... }
```

Comment:
Add some text stating what is being logged, for eg : "Intial Store state : {...}", as it would be difficult to understand what is logged when checking logs later